client/profile-wizard/steps/business-details/flows/selective-bundle/style.scss

```diff
@@ -1,4 +1,4 @@
-.woocommerce-profile-wizard__container.business-features {
+.woocommerce-profile-wizard__container.business-details {
```

Comment:
Nice!  I think we have one more reference that can be updated (removed?) - https://github.com/woocommerce/woocommerce-admin/blob/b7fad37d49962050e2c2798384015a657f24df30/client/profile-wizard/steps/business-details/style.scss#L2

Reply:
Turns out that one could actually be removed as it had the correct selector above it: https://github.com/woocommerce/woocommerce-admin/pull/8265/commits/5e57aa99a2724ca257f5fac1796cfa083b0e38e0